build/AzurePipelinesTemplates/MUX-NugetReleaseTest-Job.yml

```diff
@@ -26,13 +37,14 @@ jobs:
     packageSaveDirectory: $(Build.SourcesDirectory)\packages\muxreleasetest
     artifactDownloadPath: $(Build.SourcesDirectory)\Artifacts
     currentPackageVersion: 2.1.181217001-prerelease
+    buildOutputDir: $(Build.SourcesDirectory)\BuildOutput
```

Comment:
Update this to latest?

Reply:
What's the latest?

Reply:
2.1.190218001-prerelease right now

Reply:
Oh, I see. I thought you were talking about the buildOutputDir.